packages/chat-stateful-client/review/chat-stateful-client.api.md

```diff
@@ -21,14 +22,38 @@ export type ChatClientState = {
     threads: {
         [key: string]: ChatThreadClientState;
     };
+    latestErrors: ChatErrors;
 };
 
+// @public
+export class ChatError extends Error {
```

Comment:
Somewhat inline with @DominikMe suggestion, decided we do want to include the target that failed in the rethrown exception. But didn't go all the way in defining what the error internals look like, since we don't control that and the internals different greatly in Chat vs Calling.